### PR TITLE
fix: adjust styles for mobile and specifically for ios safari

### LINKED ISSUE #187: Possible Regression: Mobile Users should be able to access MetaMask Mobile from other mobile browser
As a user on a mobile phone

When I select Metamask wallet 

I should be able to open a link which opens the metamask app and the dapp i am on dapp inside metamask's browser

I should NOT see text telling me I dont have metamsk just because its not in window.ethereum

### PR DESCRIPTION
**Pull request by nicolasbrugneaux**
This solves two issues:

- on mobile the search bar at the bottom wasn't ideal
- on safari, 100vh isn't well supported [without a couple hacks](https://allthingssmitty.com/2020/05/11/css-fix-for-100vh-in-mobile-webkit/)
- fix #187 

Thanks @aaronmgdr for discovering this!

### CODE REVIEW

**Comment by aaronmgdr** [comment]
For me on mobile the vertical space between Search for a Wallet and the first wallet is a bit tight should be at least as much as the visual space between wallets.

**Comment by nicolasbrugneaux** [comment]
> For me on mobile the vertical space between Search for a Wallet and the first wallet is a bit tight should be at least as much as the visual space between wallets.

Done and fixed the merge conflict from the use-contractkit->react-celo :)

File: packages/react-celo/src/constants.tsx [hunk 1]
```diff
@@ -163,7 +163,7 @@ export const PROVIDERS: {
       </>
     ),
     icon: METAMASK,
-    canConnect: () => isEthereumFromMetamask(),
+    canConnect: () => true,
```
Comment (aaronmgdr) [inline packages/react-celo/src/constants.tsx:166]: this could also be `isMobile ||  isEthereumFromMetamask`

**Review by aaronmgdr — APPROVED** [review]

File: packages/react-celo/src/constants.tsx (hunk 1 above)
Comment (nicolasbrugneaux) [reply packages/react-celo/src/constants.tsx:166]: Good point.

**Pull request merged**